packages/boot/tools/supports.ts

```diff
@@ -619,36 +615,37 @@ export const makeSwingsetTestKit = async (
 
   const getCrankNumber = () => Number(kernelStorage.kvStore.get('crankNumber'));
 
-  const getOutboundMessages = (bridgeId: string) =>
-    harden([...outboundMessages.get(bridgeId)]);
-
-  /**
-   * @param {number} max the max number of messages to flush
-   * @returns {Promise<number>} the number of messages flushed
-   */
-  const flushInboundQueue = async (max: number = Number.POSITIVE_INFINITY) => {
-    console.log('🚽');
-    let i = 0;
-    for (i = 0; i < max; i += 1) {
-      const args = inboundQueue.shift();
-      if (!args) break;
-
-      await runUtils.queueAndRun(() => inbound(...args), true);
-    }
-    console.log('🧻');
-    return i;
+  const bridgeUtils = {
+    pushInbound,
```

Comment:
thinking about this more, it would be weird for a test to do this. The queue is just for mocks that are auto-generating inbound messages. The test needs a way to choose when those are processed.

If it ever wants to push a message, it should just push it at the point it wants it to be processed, rather than queuing it and flushing the queue again. The test can be responsible for sequencing with messages that are in the queue.

Reply:
This makes sense to me, it makes it more clear how these inbounds come in and would allow us to collocate mocks with the test. A potential tradeoff is more verbosity in test code, but likely not that much.